docs/manual/guide/production/ConductR.md

```diff
@@ -26,63 +48,92 @@ The above creates what is known as a "bundle". A bundle is the unit of deploymen
 
 ## Loading and running your services during development
 
-From a development perspective ConductR provides a "sandbox" environment where you can start it and then test your bundle long before it goes to production. ConductR's sandbox also permits you to [debug Lagom services](https://github.com/typesafehub/sbt-conductr-sandbox#debugging-application-in-conductr-sandbox) from your IDE when running within its cluster.
+In the previous [Installing ConductR sandbox](#Installing-ConductR-sandbox) section we have already installed the `conductr-cli`. The CLI can be used on the terminal to communicate with the ConductR cluster. 
 
-To access ConductR's sandbox from within the activator console first added the following sbt plugin to your `project/plugins.sbt`:
+To use the same commands within the activator console add the following sbt plugin to your `project/plugins.sbt`:
 
 ```scala
 addSbtPlugin("com.typesafe.conductr" % "sbt-conductr-sandbox" % "1.4.2")
 ```
 
-Additionally it is necessary to install the [conductr-cli](https://github.com/typesafehub/conductr-cli). The CLI is used by the sandbox to communicate with the ConductR cluster. Follow the "Getting Started" section on the [ConductR Developer](https://www.lightbend.com/product/conductr/developer) page to install the conductr-cli.
-
-To start the sandbox from within the console you also need to specify the ConductR version. The latest version can be picked up from the [ConductR Developer](https://www.lightbend.com/product/conductr/developer) page as well:
+To start the sandbox from within the console you also need to specify the ConductR version. Please visit the [ConductR Developer page](https://www.lightbend.com/product/conductr/developer) again to pick up the latest ConductR version from the section **Quick Configuration**:
 
 ```console
 > reload
-> set SandboxKeys.imageVersion in Global := "YOUR_CONDUCTR_SANDBOX_VERSION"
-> sandbox run
+> set SandboxKeys.imageVersion in Global := "YOUR_CONDUCTR_VERSION"
+> sandbox run --with-features logging
 ```
 
-With the sandbox running you can now load the bundle that you previously generated:
+The `reload` command loads the previously added `sbt-conductr-sandbox` plugin to your project. The `SandboxKeys.imageVersion` key is used by the `sandbox run` command to start a specific version of the the ConductR cluster locally. In this case we are starting the sandbox with the logging feature in order to retrieve logging information of the bundles.
```

Comment:
Isn't it simpler for users to add `SandboxKeys.imageVersion in Global := "YOUR_CONDUCTR_VERSION"` in their build file? (otherwise they need to remember doing it every time, and it seems like it something that is easy to forget)